go.mod

```diff
@@ -1,6 +1,7 @@
 module sigs.k8s.io/security-profiles-operator
 
 go 1.21.0
+toolchain go1.23.2
```

Comment:
We can't use go 1.23 right now, because the go-toolset does not have it: https://catalog.redhat.com/software/containers/rhel8/go-toolset

Reply:
The Kubernetes API that supports AppArmor Profile starts from version 0.30.0, which requires Go 1.22. However, the Go toolset only supports up to Go 1.21. This creates a problem, as I can’t use the necessary API while staying within the supported Go version